### ISSUE TITLE
Inventory plug-in fails if fetch_hosts_from_subcompartments not specifically set true/false; docs say default is true

### ISSUE THREAD

**Issue by jeliker** [oracle/oci-ansible-collection#25]
## Issue Report

**Describe the issue**

Inventory plug-in fails with WARNING if ```fetch_hosts_from_subcompartments``` is not specified. Docs say default for this parameter is ```true```

**Expected behavior**

I expect inventory file containing ```compartments``` list will respect the documented default ```fetch_hosts_from_subcompartments=true```

**Environment**

* OS version:
macOS 10.15.7

* Ansible version:

ansible 2.9.13
  config file = /Users/userid/oci/ansible.cfg
  configured module search path = ['/Users/userid/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/Cellar/ansible/2.9.13/libexec/lib/python3.8/site-packages/ansible
  executable location = /usr/local/bin/ansible
  python version = 3.8.5 (default, Jul 21 2020, 10:48:26) [Clang 11.0.3 (clang-1103.0.32.62)]

* OCI Python SDK version:

3.8.5

* OCI Ansible Modules version:

2.9

**Ansible playbook to reproduce the issue**

This fails in ```inventory.oci.yml```

```
compartments:
  - compartment_ocid: ocid1.compartment.oc1..xxxxxx
```

This succeeds

```
compartments:
  - compartment_ocid: ocid1.compartment.oc1..xxxxxx
    fetch_hosts_from_subcompartments: false
```

**Comment by AmeyaLokre** [comment]
Hello @jeliker , thank you for bringing this to our attention. We will be working on a fix and update this issue as soon as it is fixed.

**Comment by AmeyaLokre** [comment]
Hello @jeliker , this issue has been fixed in the new release v2.10.0 https://github.com/oracle/oci-ansible-collection/releases/tag/v2.10.0
